lib/full-site-editing/templates.php

```diff
@@ -192,3 +192,118 @@ function set_unique_slug_on_create_template( $post_id ) {
 	}
 }
 add_action( 'save_post_wp_template', 'set_unique_slug_on_create_template' );
+
+/**
+ * Inject the skip-link.
+ */
+function gutenberg_inject_skip_link() {
+	global $template_html, $skip_links_via_script;
+
+	$skip_links_via_script = array();
+
+	// Add the skip-link styles if needed.
+	if ( WP_Theme_JSON_Resolver::get_theme_data()->should_add_skip_link_styles() ) {
+		echo '<style id="skip-link-styles">';
+		echo '.skip-link.screen-reader-text{border:0;clip:rect(1px,1px,1px,1px);clip-path:inset(50%);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute !important;width:1px;word-wrap:normal !important;}';
```

Comment:
Isn't `screen-reader-text` already defined in WordPress or is it only for WP-Admin? See: https://make.wordpress.org/accessibility/2015/02/09/hiding-text-for-screen-readers-with-wordpress-core/